### PR TITLE
fix some small bugs for gamemode fw

### PR DESCRIPTION
**Pull request by LightBlueCube**
i only tested these changes on my own server running at NorthstarCN
not really sure is it working normally at here
i will very appreciate if anyone can test it
also sorry for my bad English
QQ图片20240906104002

### CODE REVIEW

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut [hunk 1]
```diff
@@ -124,6 +124,10 @@ void function GamemodeFW_Init()
 	// _battery_port.gnut needs this
 	RegisterSignal( "BatteryActivate" )
 
+	// mp_thaw's intro spawnpoint have some problem, player can be stucked on the sky, so override it
+	if( GetMapName() == "mp_thaw" )
```
Comment (ASpoonPlaysGames) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:128]:
```suggestion
	if ( GetMapName() == "mp_thaw" )
```

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut [hunk 2]
```diff
@@ -2316,7 +2322,7 @@ function FW_UseBattery( batteryPortvar, playervar ) //actually void function( en
     if( turretReplaced || teamChanged ) // replaced/hacked turret will spawn with 50% health
         newHealth = int ( turret.GetMaxHealth() * GetCurrentPlaylistVarFloat( "fw_turret_hacked_health", TURRET_HACKED_HEALTH_PERCENTAGE ) )
     // restore turret shield
-    int newShield = int ( min( turret.GetShieldHealthMax(), turret.GetShieldHealth() + ( turret.GetShieldHealth() * GetCurrentPlaylistVarFloat( "fw_turret_fixed_shield", TURRET_FIXED_SHIELD_PERCENTAGE ) ) ) )
+    int newShield = int ( min( turret.GetShieldHealthMax(), turret.GetShieldHealth() + ( turret.GetShieldHealthMax() * GetCurrentPlaylistVarFloat( "fw_turret_fixed_shield", TURRET_FIXED_SHIELD_PERCENTAGE ) ) ) )
```
Comment (ASpoonPlaysGames) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:2325]:
```suggestion
    int newShield = int( min( turret.GetShieldHealthMax(), turret.GetShieldHealth() + ( turret.GetShieldHealthMax() * GetCurrentPlaylistVarFloat( "fw_turret_fixed_shield", TURRET_FIXED_SHIELD_PERCENTAGE ) ) ) )
```

**Review by ASpoonPlaysGames — CHANGES_REQUESTED** [review]
Formatting stuff

**Review by ASpoonPlaysGames — APPROVED** [review]
Code looks good :)

**Comment by GeckoEidechse** [comment]
Why was that closed? ^^"

**Comment by LightBlueCube** [comment]
> Why was that closed? ^^"

i deleted my repo cuz i though its all finished, really sorry i didnt see the notifications
ill make a new pr or try fix this one i guess

**Pull request closed without merging**